packaging/nnstreamer.spec

```diff
@@ -5,6 +5,11 @@
 
 Name:		nnstreamer
 Summary:	gstremaer plugins for neural networks
+# Synchronize the version information among Ubuntu, Tizen, Android, and Meson.
+# 1. Ubuntu : ./jni/Android*.mk
+# 2. Tizen  : ./packaging/nnstreamer.spec
+# 3. Android: ./debian/changelog
```

Comment:
Ubuntu and android info have been swapped.

Reply:
Thanks. Fixed.